scripts/temp.deployment.sh

```diff
@@ -21,24 +22,50 @@
+cd /tmp/nls && helm dep update charts/v1.0/cray-iuf/
+cd /tmp/nls && helm package charts/v1.0/cray-iuf/
+CHART_PATH="/tmp/nls"
+
+echo "Get NLS chart version"
+tarFileName=$(ls -l "$CHART_PATH" | awk '{print $9}' | grep "cray-nls")
+tarFileName=${tarFileName#"cray-nls-"}
+version=${tarFileName%".tgz"}
+echo "Version: $version"
+
+echo "Get image version"
+imageVersion=$(yq r /tmp/nls/charts/v1.0/cray-nls/values.yaml 'cray-service.containers.cray-nls.image.tag')
+echo "image version: $imageVersion"
+echo "sync docker images"
 NEXUS_USERNAME="$(kubectl -n nexus get secret nexus-admin-credential --template {{.data.username}} | base64 -d)"
 NEXUS_PASSWORD="$(kubectl -n nexus get secret nexus-admin-credential --template {{.data.password}} | base64 -d)"
-podman run --rm --network host quay.io/skopeo/stable copy --src-tls-verify=false --dest-tls-verify=false --dest-username "$NEXUS_USERNAME" --dest-password "$NEXUS_PASSWORD"  docker://quay.io/argoproj/argocli:latest docker://registry.local/argoproj/argocli:latest
-podman run --rm --network host quay.io/skopeo/stable copy --src-tls-verify=false --dest-tls-verify=false --dest-username "$NEXUS_USERNAME" --dest-password "$NEXUS_PASSWORD"  docker://quay.io/argoproj/argoexec:latest docker://registry.local/argoproj/argoexec:latest
-podman run --rm --network host quay.io/skopeo/stable copy --src-tls-verify=false --dest-tls-verify=false --dest-username "$NEXUS_USERNAME" --dest-password "$NEXUS_PASSWORD"  docker://quay.io/argoproj/workflow-controller:latest docker://registry.local/argoproj/workflow-controller:latest
-podman run --rm --network host quay.io/skopeo/stable copy --src-tls-verify=false --dest-tls-verify=false --dest-username "$NEXUS_USERNAME" --dest-password "$NEXUS_PASSWORD"  docker://postgres:12-alpine docker://registry.local/library/postgres:12-alpine
-podman run --rm --network host quay.io/skopeo/stable copy --src-tls-verify=false --dest-tls-verify=false --dest-username "$NEXUS_USERNAME" --dest-password "$NEXUS_PASSWORD"  docker://minio/minio docker://registry.local/docker.io/minio/minio
-podman run --rm --network host quay.io/skopeo/stable copy --src-tls-verify=false --dest-tls-verify=false --dest-username "$NEXUS_USERNAME" --dest-password "$NEXUS_PASSWORD"  docker://kennethreitz/httpbin:latest docker://registry.local/docker.io/kennethreitz/httpbin:latest
-podman run --rm --network host quay.io/skopeo/stable copy --src-tls-verify=false --dest-tls-verify=false --dest-username "$NEXUS_USERNAME" --dest-password "$NEXUS_PASSWORD"  docker://portainer/kubectl-shell:latest-v1.21.1-amd64 docker://registry.local/docker.io/portainer/kubectl-shell:latest-v1.21.1-amd64
-
+podman run --rm --network host quay.io/skopeo/stable copy --src-tls-verify=false --dest-tls-verify=false --dest-username "$NEXUS_USERNAME" --dest-password "$NEXUS_PASSWORD"  docker://artifactory.algol60.net/csm-docker/stable/cray-nls:$imageVersion docker://registry.local/artifactory.algol60.net/csm-docker/stable/cray-nls:$imageVersion
```

Comment:
@mtupitsyn, is this going to break as soon as we lock down access to csm-docker?